flaml/integrations/oai/completion.py

```diff
@@ -179,18 +202,33 @@ def eval(cls, config: dict, prune=True, eval_only=False):
         """
         cost = 0
         data = cls.data
+        model = config["model"]
         data_length = len(data)
-        target_n_tokens = (
-            1000 * cls.inference_budget / cls.price1K[config["model"]]
-            if cls.inference_budget and cls.price1K.get(config["model"])
+        target_n_tokens = getattr(cls, "inference_budget", None) and (
```

Comment:
Does the `eval` API allow users to do evaluations without imposing any inference budget, which is not a rare scenario?

Reply:
Yes, it allows.